src/script/run-make.sh

```diff
@@ -56,11 +56,11 @@ function prepare() {
         if in_jenkins; then
             if ! type clang-19 > /dev/null 2>&1 ; then
                 ci_debug "Removing existing llvm packages"
-                $SUDO apt-get purge --auto-remove llvm python3-lldb -y
+                wrap_sudo
+                $DRY_RUN $SUDO apt-get purge --auto-remove llvm python3-lldb -y
```

Comment:
We will purge possibly conflicting packages only under the `if ! type clang-19` check above - Just to make sure: the machines which had 13/19 conflicts were not able to install 19 successfully and therefore would fail the !clang-19 check and are expected to enter this branch, right?

Reply:
that sounds right

Reply:
happy to move the 'apt purge' command out of the condition, but is there a way to say "uninstall all clang stuff except v19"?

or would we have to uninstall _all_ clang packages, then run llvm.sh unconditionally?